blackbox/docs/config/cluster.rst

```diff
@@ -281,6 +281,10 @@ nodes of the cluster:
      Make sure you have enough nodes and enough disk space for the
      reallocation.
 
+  .. WARNING::
+
+      This setting is deprecated and will be removed in a future release.
```

Comment:
I think we should also re-work the text above to tell the users that this setting has no effect and will be removed in the future.